app/Http/Controllers/PublicWikiController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\WikiSiteStats;
+use Illuminate\Http\Request;
+use App\Wiki;
+use App\Http\Resources\PublicWikiResource;
+
+class PublicWikiController extends Controller
+{
+    private static $defaultParams = [
+        'sort' => 'sitename',
+        'direction' => 'asc'
+    ];
+    /**
+     * Display a listing of the resource.
+     */
+    public function index(Request $request)
+    {
+        $request->validate([
+            'sort' => 'in:sitename,pages',
+            'direction' => 'in:desc,asc',
+            'is_featured' => 'boolean',
+            'is_active' => 'boolean',
+            'per_page' => 'numeric',
+            'page' => 'numeric'
```

Comment:
This is somehow magically picked up by Laravel itself, we never pass it on or look at it. Not sure if we need to validate it, but I thought it might be good to put it here so all possible query params are documented here.